### ISSUE TITLE
How to include labels during embedding

### ISSUE THREAD

**Issue by meghbhalerao** [CannyLab/tsne-cuda#86]
Hi all,
I have a very basic question. How do I incorporate the labels (for colouring the tsne plot) after the embedding has been learnt in this step: `X_embedded = TSNE(n_components=2, perplexity=15, learning_rate=10).fit_transform(X)`. From what I understand, I see only X which are the high dimensional vectors.
Thanks,
Megh

**Comment by rmrao** [comment]
tSNE does not incorporate label information, it is simply a dimensionality reduction algorithm (not a clustering algorithm). The labels are for visualization purposes only and are passed in as colors when creating a scatterplot in matplotlib.
